src/pages/Login/index.tsx

```diff
@@ -82,16 +85,21 @@ const Login: React.FC = () => {
       setAuthToken(accessToken) as unknown as void;
       setUsername('');
       setPassword('');
+      if (localStorage.getItem('mgnify.sessionExpired')) {
+        localStorage.removeItem('mgnify.sessionExpired');
```

Comment:
do we need to `await` this to avoid some race condition?

Reply:
thanks @SandyRogers actually I had meant to remove this part of the code, just pushed another commit to do that now. 
I came to the conclusion that this part is not necessary. Trying to do this kind of opened a small can of worms.
The users can always just dismiss the banner. So I thought it may not be necessary  to do this at this point in time

Reply:
Ah, perfect. Thanks!